Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+{
+  "modmenu.dropInfo.line1": "FO modlarında §9✎§r veya §8✎§r, altlarda bulunan: §c*§r;",
+  "modmenu.dropInfo.line2": "diğerleri desteklenmemektedir.",
```

Comment:
Context:

- The mods that are included as part of the Fabulously Optimized modpack have the blue or gray pencil icon before the mod options' description text. 
- Mods that may be used as an alternative by the user or during development of the pack (e.g. AdvancementInfo vs Advancement Enlarger) have a red asterisk before the main description line. 
- Mods that have neither the pencil or asterisk are not a part of Fabulously Optimized.